stdlib/public/core/String.swift

```diff
@@ -529,6 +533,7 @@ extension String {
     )
   }
 
+  @usableFromInline
```

Comment:
I had to mark this `@usableFromInline` because `_fromNonContiguousUnsafeBitcastUTF8Repairing` is marked `@_alwaysEmitIntoClient`. I'm not sure what a solution for this might look like?